clang/lib/CIR/CodeGen/CIRGenExprCXX.cpp

```diff
@@ -901,17 +901,20 @@ void CIRGenFunction::emitCXXDeleteExpr(const CXXDeleteExpr *E) {
     return;
   }
 
+  // In CodeGen:
   // We might be deleting a pointer to array.  If so, GEP down to the
   // first non-array element.
   // (this assumes that A(*)[3][7] is converted to [3 x [7 x %A]]*)
+  // In CIRGen: we handle this differently because .... .. .. . .
```

Comment:
Oh, the trailing dots were not literal haha, just a placeholder for you to mention what you need to mention :)

Reply:
Oh, my bad. Fixed.